examples/microprofile/multiport/src/main/java/io/helidon/examples/microprofile/multiport/PublicResource.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2021 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.examples.microprofile.multiport;
+
+import javax.enterprise.context.RequestScoped;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+
+/**
+ * Simple resource.
+ */
+@RequestScoped
+@Path("/public")
```

Comment:
I don't think we want to have `/public` prefixed to every public path, do we? It does emphasize that they are public paths, but that's not how an actual app would be set up.

Reply:
Yes that was done for emphasis with the assumptions folks know they can easily change this.

Reply:
Fixed. Public endpoints now default (no `/public` prefix).